src/character.cpp

```diff
@@ -132,9 +132,20 @@ bool Character::move_effects()
             remove_effect("in_pit");
         }
     }
+    if (has_effect("grabbed")){
+        if ( get_dex() > get_str() ? dice(get_dex(), 7) : dice(get_str(), 7) < get_effect_dur("downed") ){
```

Comment:
Yea, I noticed that about 10 minutes ago when I was working on the hardcoded effects. It explains a lot of my break-away-chances issues :P